src/main/resources/application.yaml

```diff
@@ -4,7 +4,7 @@ spring:
   datasource:
     url: jdbc:postgresql://localhost:5432/postgres
     username: postgres
-    password: 123
```

Comment:
Do not expose your database connection credentials. Leave the password and email empty.

Reply:
Done